### DISCUSSION TITLE
Github About 下的项目地址 是在哪里配置?

### DISCUSSION THREAD

**Discussion by vaynevayne** [umijs/dumi#1672] [Q&A] [answered]
<img width="1330" alt="image" src="https://github.com/umijs/dumi/assets/124666577/088f460a-f37e-45fe-8280-bcd4961dbb16">

请问下这个地址是在哪里配置出来的?

**Comment by afc163** [comment, marked as answer]
<img width="354" alt="图片" src="https://github.com/umijs/dumi/assets/507615/70a645d3-0f7f-42c1-9f68-3a04f2c5a319">

右上角齿轮。

**Reply by vaynevayne** [reply]
多谢
